docs/configuration.rst

```diff
@@ -77,14 +75,6 @@ below:
      - The URL to use to connect the Datadog agent. The url can starts with
        ``http://`` to connect using HTTP or with ``unix://`` to use a Unix
        Domain Socket.
-   * - ``DATADOG_TRACE_AGENT_HOSTNAME``
-     - String
-     -
-     - Deprecated: use ``DD_TRACE_AGENT_URL``
```

Comment:
Since these are deprecated I think it's worth excluding them from the documentation.